src/site/stages/build/process-cms-exports/schemas/raw/taxonomy_term-health_care_service_taxonomy.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable camelcase */
+
+module.exports = {
+  type: 'object',
+  properties: {
+    name: { $ref: 'GenericNestedString' },
+    description: {
+      oneOf: [
+        { $ref: 'GenericNestedString' },
+        {
+          type: 'array',
+          items: { type: 'object', properties: { value: { type: 'null' } } },
+        },
+      ],
+    },
+    parent: {
+      type: 'array',
+      maxItems: 1,
+      items: {
+        anyOf: [
+          { $ref: 'EntityReference' },
+          {
+            type: 'object',
+            required: ['target_id'],
```

Comment:
Were there some instances that were missing a `target_type` (and thus didn't conform to the `EntityReference` schema)?

Reply:
Yup.  I made some changes locally:

![image](https://user-images.githubusercontent.com/2008881/71844200-8690c780-307a-11ea-8017-c247188243d2.png)

And then ran the tests:

![image](https://user-images.githubusercontent.com/2008881/71844161-69f48f80-307a-11ea-8d5e-a224ad652043.png)